### PR TITLE
refactor: fix analyzer code style warnings

### PR DESCRIPTION
**Pull request by fischerscode**
### New Pull Request Checklist

- [x] I am not disclosing a [vulnerability](https://github.com/parse-community/Parse-SDK-Flutter/security/policy).
- [x] I am creating this PR in reference to an [issue](https://github.com/parse-community/Parse-SDK-Flutter/issues?q=is%3Aissue).

### Issue Description
This PR fixes the trivial, non breaking analyzer infos.

Related issue: #733 

### Approach
Mostly I just let the ide fix it.

### TODOs before merging
~~- [ ] Make sure the change to `_ParseLiveListElementWidgetState` did not break anything.~~

### CODE REVIEW

File: packages/flutter/lib/src/utils/parse_live_list.dart [hunk 1]
```diff
@@ -284,7 +283,6 @@ class _ParseLiveListElementWidgetState<T extends sdk.ParseObject>
       sizeFactor: widget.sizeFactor,
       child: AnimatedSize(
         duration: widget.duration,
-        vsync: this,
```
Comment (fischerscode) [inline packages/flutter/lib/src/utils/parse_live_list.dart:287]: I'll have to test, if this really does not break the animation.

**Comment by fischerscode** [comment]
Each `Prefer using lowerCamelCase for constant names.` info, might require a breaking change.

**Comment by fischerscode** [comment]
> Merging [#734](https://codecov.io/gh/parse-community/Parse-SDK-Flutter/pull/734?src=pr&el=desc&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=parse-community) (https://github.com/parse-community/Parse-SDK-Flutter/commit/769e90081eefa164d063ceeda7c7b963f92f8d24) into [master](https://codecov.io/gh/parse-community/Parse-SDK-Flutter/commit/1280ce3bf24305aaf24e1d61f4ca792acfcad080?el=desc&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=parse-community) (https://github.com/parse-community/Parse-SDK-Flutter/commit/1280ce3bf24305aaf24e1d61f4ca792acfcad080) will decrease coverage by 0.03%.

I think due to the introduction of `defaultParseFileConstructor`.

**Comment by mtrezza** [comment]
Kindly request a review when this is ready for merge

**Comment by fischerscode** [comment]
> I'll have to test, if this really does not break the animation.

Didn't test it yet. I'll request a review when I did.

**Comment by fischerscode** [comment]
>  Make sure the change to _ParseLiveListElementWidgetState did not break anything.

It works as expected, but I'll revert the change since it would be breaking. Older versions of Flutter (before v2.2.0-10.1.pre) require `vsync`

File: packages/flutter/lib/src/utils/parse_live_list.dart (hunk 1 above)
Comment (fischerscode) [reply packages/flutter/lib/src/utils/parse_live_list.dart:287]: It works, but I reverted it to avoid a breaking change when using older flutter versions.

**Comment by fischerscode** [comment]
codecov should be fine. Might be due to more compact format or fewer imports.

**Comment by fischerscode** [comment]
Sorry @phillwiggins pressed the wrong button.

**Review by mtrezza — CHANGES_REQUESTED** [review]
Could you please add the changelog entries to the corresponding packages?

Entries should only go to the package in which the changes occur. We do separate releases and versioning now. See https://github.com/parse-community/Parse-SDK-Flutter/issues/668#issuecomment-1140499834

If this PR does not fix a bug, add a feature or improve performance, but is merely a refactor to get rid of recommended code style changes, then this would not lead to a changelog entry. It would be merge as something like `refactor: fix analyzer code style warnings`.

File: packages/dart/CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,10 @@
 * General improvements
 * Updated dependencies
 
+### Refactoring
```
Comment (mtrezza) [inline packages/dart/CHANGELOG.md:8]: Could you move this to the top under a new version 3.1.1?
And rename the title to "Refactor".

File: packages/flutter/CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,10 @@
 
 * update example app to use Android embedding v2 ([#722](https://github.com/parse-community/Parse-SDK-Flutter/issues/722)) ([e092189](https://github.com/parse-community/Parse-SDK-Flutter/commit/e092189cb666c25b3e2c9dbbf95316e9cfa88e72))
 
+### Refactoring
```
Comment (mtrezza) [inline packages/flutter/CHANGELOG.md:7]: Could you move this to the top under a new version 3.1.2?
And rename the title to "Refactor".

**Review by mtrezza — CHANGES_REQUESTED** [review]

**Review by mtrezza — APPROVED** [review]
Looks good! Let's try out this first dart - flutter combo release!

**Comment by mtrezza** [comment]
Oops, we forgot forgot to bump the package versions. So no release this time :-) We'll just wait for the next PR, probably https://github.com/parse-community/Parse-SDK-Flutter/pull/738, and do a release.

**Comment by fischerscode** [comment]
> Oops, we forgot forgot to bump the package versions.

Oops...

> Unless you feel like opening another PR just for the version bump.

If you want, I can do it. It might be best to test it soon rather then later.

**Comment by mtrezza** [comment]
Sure, please go ahead

**Comment by fischerscode** [comment]
> Sure, please go ahead

Link it to #733?

**Pull request merged**